lib/hterm.js

```diff
@@ -230,7 +230,8 @@ hterm.Keyboard.prototype.onKeyDown_ = function (e) {
     e.preventDefault();
   }
   // hterm shouldn't consume a hyper accelerator
-  if (e.altKey || e.metaKey || isCommands(e)) {
+  const isCmd = isCommands(e);
```

Comment:
This function call returns if it is a Command, but execute plugin command.
Name is a little ambiguous.

this is a good thing to prevent hterm to consume a command. But maybe key detection and callback execution should not rely on hterm. We want that our keys work without a focused hterm (webview, other pane type etc..).